man/man1/slice2matlab.1

```diff
@@ -80,8 +73,7 @@ when this option is specified.
 .TP
 .BR \-\-depend\-file " " FILE\fR
 .br
-Directs dependency information to the specified file. The output
-format depends on whether --depend or --depend-xml is also specified.
+Directs dependency information to the specified file.
```

Comment:
We should mention that this is useless until you also pass `--depend-xml`.

Reply:
Fixed